.changeset/feat-418-listjobs-agents-filter.md

```
@@ -0,0 +1,46 @@
+---
+"@herdctl/core": minor
+---
+
+feat(state): filter `listJobs` by a SET of agents, and page the dashboard's `/api/jobs` inside it
+
+#415 made `listJobs` index-backed, but only for callers that can express what
```

markdownlint-cli2 (0.23.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)